spec/factories/in_person_enrollments.rb

```diff
@@ -43,7 +43,11 @@
     end
 
     trait :with_sponsor_id do
-      sponsor_id { '123458' }
+      sponsor_id  { IdentityConfig.store.usps_ipp_sponsor_id }
+    end
```

Comment:
@eileen-nava  Is there a reason I should not use config here and keep a random string?

Reply:
I will investigate and get back to you on that question. 👍🏻

Reply:
Okay, so here is what I have uncovered by testing.... 

1. If we hard code the number here- enrollments using this bot will always have a sponsor_id
2. If we don't hard code the number and use the config store, it will use the value passed into the spec when we mock it. 
    allow(IdentityConfig.store).to receive(:usps_ipp_sponsor_id).and_return(usps_ipp_sponsor_id)
    allow(IdentityConfig.store).to receive(:usps_eipp_sponsor_id).and_return(usps_ipp_sponsor_id)
3. If we don't hard code the number and use the config store, if there is no value passed into the spec it will be nil so sponsor id on enrollment will not be set. It looks like I can set `usps_ipp_sponsor_id` in application.yml.default in the test section and then for all of the specs without the mock setup- it will return that value. 

I feel like using the config store above and adding a default config in application.yml.default is the best path forward. I am still testing.

Reply:
> It looks like I can set usps_ipp_sponsor_id in application.yml.default in the test section and then for all of the specs without the mock setup- it will return that value.

I agree with this path forward.  In response to the UPDATE, I also agree that using the config is better than hardcoding.